staff/joseluis-juste/app-template/about.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>ABOUT</title>
+</head>
+
+<body>
+    <header>
+        <img src="https://picsum.photos/200/200/?" alt="">
+    </header>
+    <nav>
+        <a href="about.html">About</a>
+        <a href="contact.html">Contact</a>
+        <a href="login.html">Login</a>
+        <a href="news.html">News</a>
+        <a href="profile.html">Profile</a>
+        <a href="home.html">Home</a>
+        <a href="register.html">Register</a>
+    </nav>
+
+    <aside>
+        <ul>
+            <li><a href="about.html">About</a></li>
+            <li><a href="contact.html">Contact</a></li>
+            <li><a href="login.html">Login</a></li>
+            <li><a href="news.html">News</a></li>
+            <li><a href="profile.html">Profile</a></li>
+            <li><a href="home.html">Home</a></li>
+            <li><a href="register.html">Register</a></li>
+        </ul>
```

Comment:
why there's a duplicated menu? btw in the first you left the ul/li structure and in the second you left the nav tag

Reply:
I know and in a real app i wouldn´t do it. I put it there only for fill with some of information

Reply:
remove it, as it breaks good practices. semantics should be clear, but content too.

Reply:
Ok, It has been fixed